estimate-web/src/components/UserStoryCommentEditor.less

```diff
@@ -1,6 +1,10 @@
 @import "../resources/Measurements.less";
+@import "../resources/Borders.less";
 
 .user-story-comment-editor {
+    .ms-TextField {
```

Comment:
I'm curious how is this different from `.user-story-comment-editor.ms-TextField` which I used in a similar situation

Reply:
It would just allow me to add extra rules to the "parent" css class `.user-story-comment-editor`
it's just a LESS shorthand